config.test.yaml

```diff
@@ -64,6 +54,27 @@ templates:
           internal:
             - /^(?:::ffff:)?(?:10|127)\./
 
+    x-modules:
+      - name: mobileapps-public
+        path: specs/mediawiki/v1/mobileapps
+        type: spec
+        options:
+          host: http://appservice.wmflabs.org
+
+      - name: graphoid
+        path: mods/restbase/graphoid_v1
```

Comment:
Why `restbase` and not `wikimedia`? Also, IMHO it'd be better to have `mods/wikimedia/v1/graphoid` (idem for summary below).